src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -871,54 +871,91 @@ public void stopService() {
     // release the interpeter
     stop();
   }
+  
+  public void init() {
+    
+    log.info("created python {}", getName());
+    createPythonInterpreter();
+    sleep(250);

```

Comment:
What's this sleep for? Should have at least a comment explaining it, hopefully it can be removed eventually

Reply:
its asynchronous initialization ... if i remember correctly :-1: